command/client_config.go

```diff
@@ -7,20 +7,20 @@ import (
 	"github.com/posener/complete"
 )
 
-type ClientConfigCommand struct {
+type NodeConfigCommand struct {
 	Meta
 }
 
-func (c *ClientConfigCommand) Help() string {
+func (c *NodeConfigCommand) Help() string {
 	helpText := `
-Usage: nomad client-config [options]
+Usage: nomad node config [options]
 
-  View or modify client configuration details. This command only
-  works on client nodes, and can be used to update the running
-  client configurations it supports.
+  View or modify a client node's configuration details. This command only works
+  on client nodes, and can be used to update the running client configurations
+  it supports.
 
-  The arguments behave differently depending on the flags given.
-  See each flag's description for its specific requirements.
+  The arguments behave differently depending on the flags given. See each
+  flag's description for its specific requirements.
```

Comment:
Should this old alias `client config` be mentions in the docs ?

Reply:
I only called out aliases we want to keep. Like shortcuts. We will need to add a version section to: https://www.nomadproject.io/docs/upgrade/upgrade-specific.html